### PR TITLE
Bump buildpacks

### PR DESCRIPTION
**Pull request by titanous**
Also added a handy tool for doing this automatically.

https://github.com/heroku/heroku-buildpack-ruby/compare/v129...20b99c8c
https://github.com/heroku/heroku-buildpack-nodejs/compare/1ebb095a...197845da
https://github.com/heroku/heroku-buildpack-clojure/compare/v63...90fbff0d
https://github.com/heroku/heroku-buildpack-python/compare/v54...532b36f5
https://github.com/heroku/heroku-buildpack-java/compare/e99a5f80...cfdb4b99
https://github.com/heroku/heroku-buildpack-gradle/compare/24a8ebe7...47dc93f2
https://github.com/heroku/heroku-buildpack-scala/compare/f5415da1...90124426
https://github.com/heroku/heroku-buildpack-play/compare/9c137b4a...1cf304b4
https://github.com/heroku/heroku-buildpack-php/compare/v51...1746ff03
https://github.com/kr/heroku-buildpack-go/compare/3a2cc0fa...daef77e9

### CODE REVIEW

**Comment by archSeer** [comment]
LGTM

File: util/bump-buildpacks/bump.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		log.Println("no updates")
+		os.Exit(0)
+	}
+
+	if err := ioutil.WriteFile(dataFile, out.Bytes(), 0644); err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Fprintf(commitMsg, "\nSigned-off-by: %s <%s>\n", gitConfig("user.name"), gitConfig("user.email"))
+	cmd := exec.Command("git", "commit", "-m", commitMsg.String(), "--", dataFile)
```
Comment (lmars) [inline util/bump-buildpacks/bump.go]: Could just add `--signoff` to the command line rather than manually adding the sign off to the commit message above

**Comment by lmars** [comment]
LGTM pending minor comment (I probably would've wrote a bash script for the bump-buildpacks tool :trollface:)

File: util/bump-buildpacks/bump.go (hunk 1 above)
Comment (titanous) [inline util/bump-buildpacks/bump.go]: Tried that, but it strips the empty newline between the message body and the added sign-off line.

Comment (lmars) [inline util/bump-buildpacks/bump.go]: Seems to work for me:

```
$ git --version
git version 1.9.3 (Apple Git-50)

$ git commit --allow-empty --signoff -m "$(echo -e "foo\n\nbar\nbaz")" 

$ git show --pretty=format:%B
foo

bar
baz

Signed-off-by: Lewis Marshall <lewis@lmars.net>
```

That being said, feel free to merge.

Comment (titanous) [inline util/bump-buildpacks/bump.go]: Yeah, I was probably doing it wrong.

**Pull request merged**